content/en/docs/tasks/administer-cluster/configure-feature-gates.md

```diff
@@ -0,0 +1,198 @@
+   - **Alpha**: Disabled by default, may be buggy. Use only in test clusters.
+   - **Beta**: Usually enabled by default, well-tested.
+   - **GA**: Always enabled, cannot be disabled.
+
+2. Determine which components need the feature gate enabled:
+   - Some features require enabling the gate on **multiple components** (e.g., API server and controller manager)
+   - Other features only need the gate on a **single component** (e.g., only kubelet)
+
+3. Use the component's help flag to verify it supports the feature gate:
+
+   ```shell
+   kube-apiserver -h | grep feature-gates
+   kubelet -h | grep feature-gates
+   ```
+
+{{< note >}}
+Each Kubernetes component only accepts feature gates relevant to its functionality. 
+The Feature Gates reference page typically indicates which components are affected by each gate.
+{{< /note >}}
+
+## Configure for kubeadm clusters
+
+### During cluster initialization
+
+Create a configuration file to enable feature gates across relevant components:
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta4
+kind: ClusterConfiguration
+apiServer:
+  extraArgs:
+    feature-gates: "FeatureName=true"
+controllerManager:
+  extraArgs:
+    feature-gates: "FeatureName=true"
+scheduler:
+  extraArgs:
+    feature-gates: "FeatureName=true"
+---
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+featureGates:
+  FeatureName: true
+```
+
+Initialize the cluster:
+
+```shell
+kubeadm init --config kubeadm-config.yaml
+```
+
+### On an existing cluster
+
+Edit control plane component manifests in `/etc/kubernetes/manifests/`:
+
+1. For kube-apiserver, kube-controller-manager, or kube-scheduler, add the flag to the command:
+
+   ```yaml
+   spec:
+     containers:
+     - command:
+       - kube-apiserver
+       - --feature-gates=FeatureName=true
+       # ... other flags
+   ```
+
+   Save the file. The pod restarts automatically.
+
+2. For kubelet, edit `/var/lib/kubelet/config.yaml`:
+
+   ```yaml
+   apiVersion: kubelet.config.k8s.io/v1beta1
+   kind: KubeletConfiguration
+   featureGates:
+     FeatureName: true
+   ```
+
+   Restart kubelet:
+
+   ```shell
+   sudo systemctl restart kubelet
+   ```
+
+3. For kube-proxy, edit the ConfigMap:
+
+   ```shell
+   kubectl -n kube-system edit configmap kube-proxy
+   ```
+
+   Add feature gates to the configuration:
+
+   ```yaml
+   featureGates:
+     FeatureName: true
+   ```
+
+   Restart the DaemonSet:
+
+   ```shell
+   kubectl -n kube-system rollout restart daemonset kube-proxy
+   ```
+
+## Configure multiple feature gates
+
+Use comma-separated lists for the command-line flag:
+
+```shell
+--feature-gates=FeatureA=true,FeatureB=false,FeatureC=true
+```
+
+Or in YAML configuration files:
+
+```yaml
+featureGates:
+  FeatureA: true
+  FeatureB: false
+  FeatureC: true
+```
+
+<!-- discussion -->
+
+## Verify feature gate configuration
+
+After configuring, verify the feature gates are active.
+
+For control plane components:
```

Comment:
This only applies if a control plane component is deployed as a static pod.
The reality is much more complicated than this.